openwisp_users/migrations/0003_default_organization.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 2.1.1 on 2018-09-16 12:20
+
+from django.db import migrations
+from django.conf import settings
+
+
+def set_default_organization_uuid(apps, schema_editor):
+    """
+    Get default organzation or create a default
+    organization and set settings.DEFAULT_ORG_UUID
+    """
+    organization = apps.get_model('openwisp_users', 'organization')
+    try:
+        default_organization = organization.objects.get(
+            slug=settings.DEFAULT_ORG_SLUG)
+
+    except (AttributeError, organization.DoesNotExist):
+        default_organization = organization.objects.create(
+            name='default',
+            slug='default',
+            is_active=False,
+            description='This is the default organization. '
+            'It created because no organization was set as default '
+            'during setup. If you want to delete it, please make sure '
+            'all the users are moved to appropriate organization '
+            'because when an organization is delete all the '
+            'organization users are deleted as well!',
```

Comment:
I like this idea. I would change the hint and tell them that instead of deleting it they should simply rename it.